conu/apidefs/backend.py

```diff
@@ -0,0 +1,53 @@
+"""
+Definition for a backend class
+"""
+
+import logging
+
+from .container import Container
```

Comment:
This should be `DockerContainer` and not `Container`, right?

Reply:
no, this is the common interface which is shared for all the implementations